src/main/java/xyz/vegaone/easytrackingv2/repo/ProjectRepo.java

```diff
@@ -9,4 +10,7 @@
 public interface ProjectRepo extends JpaRepository<ProjectEntity, Long> {
 
     List<ProjectEntity> findAllByUserList(List<UserEntity> userList);
+
+    @Query("SELECT new xyz.vegaone.easytrackingv2.domain.ProjóectEntity (id, name)  FROM  project")
+    List<ProjectEntity> findAllProjectsAndDisplayProjectIdAndName();
```

Comment:
A more appropriate name for this method would be `findAllProjectsBrief()`, as we want to fetch only a small part of the object.
And there is a weird character in the query: `ProjóectEntity`